spring-boot-samples/spring-boot-sample-kafka/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<!-- Your own application should inherit from spring-boot-starter-parent -->
+		<artifactId>spring-boot-samples</artifactId>
+		<groupId>org.springframework.boot</groupId>
+		<version>${revision}</version>
+	</parent>
+	<artifactId>spring-boot-sample-kafka</artifactId>
+	<name>Spring Boot Kafka Sample</name>
+	<description>Spring Boot Kafka Sample</description>
+
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
+
+	<dependencies>
+		<!-- Compile -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.kafka</groupId>
+			<artifactId>spring-kafka</artifactId>
```

Comment:
There is a kafka starter, please use that instead.

Reply:
Where can i find the kafka starter?

Reply:
My bad, I thought we had a Kafka starter but we don't. This one is good.

Reply:
Ok, Thanks.

Reply:
Hi, i have reworked, follow other samples, please check.